cli/net_test.go

```diff
@@ -0,0 +1,272 @@
+//stm: #cli
+package cli
+
+import (
+	"context"
+	"testing"
+
+	"github.com/filecoin-project/lotus/api"
+	metrics "github.com/libp2p/go-libp2p-core/metrics"
+	"github.com/libp2p/go-libp2p-core/peer"
+	protocol "github.com/libp2p/go-libp2p-core/protocol"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestNetPeers(t *testing.T) {
+	peerAddrInfo := []peer.AddrInfo{}
+	app, mockApi, _, done := NewMockAppWithFullAPI(t, WithCategory("net", NetPeers))
+	defer done()
+
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	mockApi.EXPECT().NetPeers(ctx).Return(peerAddrInfo, nil)
```

Comment:
Those tests should at least test some output, just calling the commands and seeing no error with no output is not that useful by itself.